src/CalcManager/Ratpack/conv.cpp

```diff
@@ -206,16 +209,18 @@ PNUMBER _createnum( _In_ uint32_t size )
     PNUMBER pnumret= nullptr;
     uint32_t cbAlloc;
```

Comment:
Azure complains:
`src\calcmanager\ratpack\conv.cpp(212,0): Error C4700: uninitialized local variable 'cbAlloc' used`

Reply:
I know, working on it..